doc/architecture.md

```diff
@@ -97,7 +97,7 @@ repositories encourages the use of divergent tooling and silos.
 
 Most organizations do not have a mono-repo because existing DVCS systems (e.g.,
 Git and Mercurial) suffer performance degradation as the size of the repository
-and the number of users increase.  Over time, basic operations such as `git
+and the number of users increases.  Over time, basic operations such as `git
```

Comment:
I think this change is not needed, since both the size and the number "increase".
Please "git commit --amend" this commit locally to remove this change, then "git push -f" the branch to update this pull request.

Reply:
I just updated my pull request based on the feedback you gave me.